library/CMakeLists.txt

```diff
@@ -60,8 +60,8 @@ add_subdirectory( src )
 # endif( )
 
 # Package specific CPACK vars
-set( CPACK_DEBIAN_PACKAGE_DEPENDS "rocblas (>= 0.10.3)" )
-set( CPACK_RPM_PACKAGE_REQUIRES "rocblas >= 0.10.3" )
+set( CPACK_DEBIAN_PACKAGE_DEPENDS "rocblas (>= 2.28.0), rocsolver (>= 3.8.0)" )
+set( CPACK_RPM_PACKAGE_REQUIRES "rocblas >= 2.28.0, rocsolver >= 3.8.0" )
```

Comment:
Should this be == instead of >=. We do not support mixing and matching currently.

Reply:
How would setting it to == play out? It seems to me that it would cause issues during the development cycle, especially if we're requiring a certain version here on the develop branch. If version `x` gets released tomorrow, would we then want to update this to `== x` immediately? What would it look like when moving to release branches?
I think that documenting that mixing versions isn't supported is enough, I'm not sure it has to be enforced here.

Reply:
We won't plan on taking things out of the API so >= to me makes sense.  Hipblas can likely work with newer version of rocsolver, but rocsolver can likely only work with one version of rocBLAS.  It could be we only depend on rocSOLVER as that will pull in rocBLAS, unless it is being built as without solver support  :-(

Reply:
If rocsolver is tightly coupled with a specific rocblas version, I think we should enforce it in the packaging for rocsolver. hipblas can depend on rocsolver to provide rocblas. This may head off future user complaints. Just my suggestion. Ideally, rocsolver wouldn't be tightly coupled with rocblas, but we should enforce what is true now.